icu4c/source/data/BUILDRULES.py

```diff
@@ -517,18 +516,18 @@ def generate_tree(
     ]
 
     # Generate res_index file
-    synthetic_locales = set()
-    if not config.ignore_xml_deprecates:
```

Comment:
I hadn't seen this `ignore_xml_deprecates` before. Is v8 using it now?

Reply:
I added it in #642 and #646 (post-64) but I hadn't formally proposed it at the time, and now I'm bringing it up and we're figuring out what that feature should look like.